pyproject.toml

```diff
@@ -45,6 +45,7 @@ alembic = "^1.12.0"
 atomicwrites = "^1.2.1"
 azure-common = {version = "^1.1.28", optional = true}
 azure-core = {version = "^1.29.4", optional = true}
+azure-identity = {version = "^1.14.1", optional = true}
```

Comment:
This was missing so the dependency wasn't present in the lock file. The library actually does have type annotations.